reference/replica-operations.md

```diff
@@ -32,45 +29,42 @@ N.B.: By application of the 2nd rule, replicas of the same volume cannot exist w
 
 ### Scenario Two
 
 A cluster has three MSNs deployed, "Node-1", "Node-2" and "Node-3".  Each MSN hosts one MSP: "Pool-1" on Node-1, "Pool-2" on Node-2 and "Pool-3" on Node-3.  No MSVs have yet been defined; the cluster is 'clean'.  A user creates a PVC from a StorageClass which defines a replica count of 2.  The control plane determines that it is possible to accommodate one replica within the available capacity of each of Pool-1 and Pool-2, and so the MSV is created.  An application is deployed on the cluster which uses the PVC, so the volume receives I/O.
 
-Unfortunately, due to user error the SAN LUN which is used to persist Pool-2 becomes detached from Node-2, causing I/O failures in the replica which it hosts for the volume.  As with scenario one, the volume state becomes `degraded` and the faulted replica's becomes `CHILD_FAULTED`.
+Unfortunately, due to user error the SAN LUN which is used to persist Pool-2 becomes detached from Node-2, causing I/O failures in the replica which it hosts for the volume.  As with scenario one, the volume state becomes `Degraded` and the faulted child's becomes `Faulted`.
 
-Expected Behaviour: Since there is a MSP on Node-3 which has sufficient capacity to host a replacement replica, a new replica can be created (Rule 2: this 'third' incoming replica isn't located on either of the MSNs that the two original ones are).  A new replica is created on Pool-3 and added to the nexus, and the faulted replica in Pool-2 is retired from the nexus.  The new replica is rebuilt and eventually the state of the MSV returns to `healthy`.
+Expected Behaviour: Since there is a MSP on Node-3 which has sufficient capacity to host a replacement replica, a new replica can be created (Rule 2: this 'third' incoming replica isn't located on either of the MSNs that the two original ones are). The faulted replica in Pool-2 is retired from the nexus and a new replica is created on Pool-3 and added to the nexus.  The new replica is rebuilt and eventually the state of the MSV returns to `Online`.
 
 ### Scenario Three
 
-In the cluster from Scenario three, sometime after the MSV has returned to the `healthy` state, a user edits the MSV's `Replica Count:` field, increasing the value from 2 to 3.  Before doing so they corrected the SAN misconfiguration and ensured that the MSP on Node-2 was `online`.
+In the cluster from Scenario three, sometime after the MSV has returned to the `Online` state, a user scales up the MSV, increasing the `num_replicas` value from 2 to 3.  Before doing so they corrected the SAN misconfiguration and ensured that the MSP on Node-2 was `Online`.
```

Comment:
They need to use the kubectl plugin do to this, yes?  So no MSV? MSV->PV ?

Reply:
Correct, shall I allude to that? eg: (MSV has returned to `healthy` (as observed in the volume through the kubectl-mayastor plugin)).
Since we're here, through the document we used the MSV/MSN nomenclature to refer to volumes and nodes, should we keep it for simplicity or call them something else?

Reply:
Good question.  I've had the same conversation with Anupriya with regards to Mayastor node.  Whilst that's no longer a valid reference to a k8s CRD, it's still a distinct entity within a Mayastor deployment.  We agreed to refer to it whenever necessary as "Mayastor node", not MSN (to try to avoid any confusion with CRs in previous versions).

Here, I think what we're talking about is a Mayastor volume but I'm inclined to go with just "volume" or "persistent volume", as that's what it really is in more generic k8s terms.  Open to your suggestions.

Reply:
We can introduce it as "Mayastor volume" when introducing it and then simply refer to "volume" for brevity? (i.e the volume becomes `Online`) ?

Reply:
I agree.